strawberry/schema/schema_converter.py

```diff
@@ -340,7 +344,7 @@ def get_graphql_input_fields(
         )
 
     def from_input_object(self, object_type: type) -> GraphQLInputObjectType:
```

Comment:
Just notice that `from_input_object` is the only converter that receives a type instead of `StrawberryObjectDefinition`

@patrick91 should we try to normalize this? Don't know if in this PR, but since the breaking change is happening it would be nice. Either this function should receive the object definition, like the others (e.g. from_object, from_interface, etc), or those others should receive the `type` directly (even typing it as `type: WithObjectDefinition`)

Reply:
let's do in another PR 😊